### DISCUSSION TITLE
Ripetitore Zigbee IKEA

### DISCUSSION THREAD

**Discussion by ElyFNV** [indomus/forum#937] [HUB: Home Assistant]
Ciao a tutti!
Ho alcuni rilevatori di movimento Philips e alcuni sensori di temperatura collegati ad Home Assistant tramite USB ConBee II. Tutti ovviamente usano il protocollo Zigbee. Tuttavia vorrei aggiungere altri rilevatori di movimento Zigbee in alcune stanze, purtroppo troppo lontani perché il segnale raggiunga correttamente l'usb ConBee collegato al Raspberry. Così ho comprato questo ripetitore zigbee IKEA (https://www.ikea.com/it/it/p/tradfri-ripetitore-di-segnale-10400408/).
Ma la mia domanda adesso è: devo collegare in qualche modo il ripetitore Ikea a Home Assistant ed integrarlo? O è sufficiente collegarlo alla rete elettrica per poter estendere il segnale Zigbee?
Mi spiego: una volta collegato il ripetitore Ikea alla presa elettrica, la rete zigbee esistente si estende automaticamente? E quindi posso collegare i sensori di movimento Zigbee (lontani dal Raspberry) senza problemi? Oppure devo prima integrare in qualche modo il ripetitore Ikea in Home Assistant?
Grazie mille a tutti!!

**Comment by sky7176** [comment]
Ciao,
devi in ogni caso associare il ripetirore alla tua rete, dopodiche essendo la rete Zigbee di tipo mesh, in tutta autonomia saranno i vari dispositivi che sceglieranno il path migliore per collegardi al coordinator.
Considera anche che tutti i dispositivi non a batteria sono dei ripetirori Zigbee per default.

Giuseppe

**Reply by angelopampalone** [reply]
> Ciao, devi in ogni caso associare il ripetirore alla tua rete, dopodiche essendo la rete Zigbee di tipo mesh, in tutta autonomia saranno i vari dispositivi che sceglieranno il path migliore per collegardi al coordinator. Considera anche che tutti i dispositivi non a batteria sono dei ripetirori Zigbee per default.
> 
> Giuseppe

Mi permetto di intromettermi; Vorrei al conferma che le lampade Tradfri Ikea fanno da ripetitori Zigbee.
Zigbee2Mqtt mi pare che nella map non le usi mai andando direttamente alla ConBee II

Grazie mille,
Angelo

**Reply by sky7176** [reply]
Ciao Angelo,
tutti i dispositivi alimentati dalla rete elettrica sono dei ripetitori, poi che vengano utilizzati come tali è un altro discorso, come ti dicevo essendo la rete Zigbee di tipo mesh sono i dispositivi che cercano il percorso a minor costo per raggiungere il coordinator.

Giuseppe

**Reply by angelopampalone** [reply]
> Ciao Angelo, tutti i dispositivi alimentati dalla rete elettrica sono dei ripetitori, poi che vengano utilizzati come tali è un altro discorso, come ti dicevo essendo la rete Zigbee di tipo mesh sono i dispositivi che cercano il percorso a minor costo per raggiungere il coordinator.
> 
> Giuseppe

Ok, evidentemente è più efficiente per loro parlare direttamente con la ConBee 2.

Grazie,
Angelo

**Reply by ElyFNV** [reply]
Fantastico! Grazie mille davvero per le risposte!
Quindi io vado a collegare il ripetitore zigbee di IKEA alla chiavetta Conbee II tramite DeConz/Phoscon come farei con qualsiasi altro sensore. E poi, una volta collegato, vado a collegare a DeConz/Phoscon tutti gli altri dispositivi (sensori) posizionati più lontani dalla chiavetta Conbee II (come ho sempre fatto)? E te mi dici che i nuovi sensori (quelli più lontani) dovrebbero collegarsi automaticamente utilizzando il ripetitore (essendo la rete Zigbee di tipo mesh)?

**Reply by snakuzzo** [reply]
sì. La connessione ai router comunque la gestisce il coordinator. Tu utente non puoi avere alcun controllo
